api/src/client/ApiFormationClient.test.ts

```diff
@@ -665,6 +671,92 @@ describe("ApiFormationClient", () => {
           },
         });
       });
+
+      describe("getPracticesLaw", () => {
+        let stubResponse: ApiResponse;
+
+        beforeEach(() => {
+          jest.resetAllMocks();
+          stubResponse = generateApiResponse({});
+          mockAxios.post.mockResolvedValue({ data: stubResponse });
+        });
+
+        it("PracticesLaw is undefined if not provided", async () => {
```

Comment:
not a big deal but just a note that we try to write test names as a sentence starting with "it..."

Reply:
The reason for this is that it's a technique to encourage behavioral testing. As a principle, we always want to be testing some _output_ or behavior of the thing under test. So if we can describe what "it" does - that's its behavior. And if there's something that we can't put in this format, it's probably not actually observable behavior and we should consider how to refactor the test